data/json/effects.json

```diff
@@ -386,6 +386,21 @@
     },
     {
         "type": "effect_type",
+        "id": "nausea",
+        "name" : ["Nausea"],
+        "desc": ["You feel nauseous.  Even a thought of food makes you feel sick."],
+        "remove_message": "You no longer feel nauseous.",
+        "rating": "bad",
+        "resist_traits": ["STRONGSTOMACH"],
```

Comment:
Why not put it as immunity, in `player::immune_effect`? That way it won't be applied at all to STRONGSTOMACH characters.